Src/Idioms/EmptyGuidBehaviorExpectation.cs

```diff
@@ -25,18 +26,37 @@ public class EmptyGuidBehaviorExpectation : IBehaviorExpectation
         /// </remarks>
         public void Verify(IGuardClauseCommand command)
         {
-            if (command == null) throw new ArgumentNullException(nameof(command));
+            if (command == null)
```

Comment:
I would not make these guard clauses multi-line. This way it clutters the code and decreases readability by distracting from an essence of the method. One liners don't suffer from the initial issue with multil-line `if` statements, when you could mistakenly read statements due to poor formatting.

I do agree though that we either use one-line style without curly braces, or with curly braces if more than one line.